yesod-test/Yesod/Test.hs

```diff
@@ -693,6 +694,25 @@ get url = request $ do
     setMethod "GET"
     setUrl url
 
+-- | Follow a redirect, if the last response was a redirect.
```

Comment:
It would be good if this said what qualified as a redirect (301 or 303 HTTP code + "Location" header in response).